README.md

```diff
@@ -35,22 +35,32 @@ I am not affiliated with reMarkable AS in anyway
 
 
 ## Quick Doc
-- bookmark pages by clicking the upper right (or left for lefthanders) corner
-- long press on a bookmark or upper left/right corner (touch not pen) to edit it
-- some menus to toggle swipes and bookmarks are in the "Share Menu" (rectangle with an arrow pointing out)
-- there is an exit button in the "Document Menu" (bottommost)
-- left & right button simultaneously to toggle zen mode
-- long press on the "toggle menu" (uppermost) to toggle zen mode
-    zen mode:
-        - left/right toggles pen/eraser
-        - left/right long press to undo
-- home & right button simultaneously to enter reading mode (use the shortcut for zen mode)
-    - tap left/right to change panges
-- swipe down to toggle the menu
-- long press home button to show Recent Files
-- long press on a recent file that was deleted but not synced to restore it
-- 2 finger swipe up for recent files
+
+Note: the reMarkable 2 does not have buttons
+
+#### Gestures
```

Comment:
I used Heading 4 instead of Heading 2 or nested bullets because the latter were not easy to read.